apps/passport-client/src/dispatch.ts

```diff
@@ -966,12 +976,36 @@ async function addSubscription(
 async function removeSubscription(
   state: AppState,
   update: ZuUpdate,
-  subscriptionId: string
+  subscriptionId: string,
+  deleteContents?: boolean
```

Comment:
nice

Reply:
should make [0XP-13](https://linear.app/0xparc-pcd/issue/0XP-13/automatically-remove-old-lemonade-pcd-if-it-exists) super trivial actually